src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -138,6 +168,12 @@ public abstract partial class WebRequestPSCmdlet : PSCmdlet
         public virtual SwitchParameter SkipCertificateCheck { get; set; }
 
         /// <summary>
+        /// Gets or sets the TLS/SSL protocol used by the Web Cmdlet
+        /// </summary>
+        [Parameter]
+        public virtual WebSslProtocol SslProtocol { get; set; } = WebSslProtocol.Default;
```

Comment:
I should ask: maybe use `WebSslProtocol` for parameter name?

Reply:
I think the `Web` is kind of redundant in the parameter name. I was just continuing with the convention of prefixing `Web` in front of Enums used for web cmdlet parameter options. Technically, it is setting the `SslProtocol`, we are just limiting the options that the user can use to those it actually supports.